movement.cpp

```diff
@@ -0,0 +1,435 @@
+#include "movement.h"
+#include "pixy.h"
+
+front_left_motor_ = Motor(frontLeftMotorPins[0], frontLeftMotorPins[1], frontLeftPwm);
+front_right_motor_ = Motor(frontRightMotorPins[0], frontRightMotorPins[1], frontRightPwm);
+back_left_motor_ = Motor(backLeftMotorPins[0], backLeftMotorPins[1], backLeftPwm);
+back_Right_motor_ = Motor(backRightMotorPins[0], backRightMotorPins[1], backRightPwm);
+
+void Movements::move0()
+{
+  front_left_motor_.changePwm(230.0);
```

Comment:
why are you always using 230?
maybe it wouls be better to implement a function that set the 4 motors the same pwm.